oss_fuzz_integration/project_checker.py

```diff
@@ -144,12 +145,30 @@ def check_specific_project(proj_name, build_log_file,coverage_log,summary_json):
     print("Check done")
 
 
+def check_project_html_sanity(html_report):
+    """Checks the sanity of URLs in the html file. Returns False if fails"""
+    with open(html_report, 'r') as html_file:
+        for line in html_file:
+            line_pos = line.find("<a href")
+            if line_pos != -1:
```

Comment:
nit: we can avoid a lot of unnecessary indentation if we just invert this condition and continue here.